client/client.go

```diff
@@ -0,0 +1,145 @@
+package main
```

Comment:
I would give the same comments as for the `cli/cli.go`

- Cyclomatic complexity has to be kept minimal, must not being ignored
- Misused exposed package variables
- Multiline strings
- os.Env must be consumed within `init()` functions where it's possible
- error variables and checks would be kept short for conditional blocks where is possible, e.g.: 
```
if err :=...; err != nil {
		...
}
```